src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,117 @@
+@import '/src/prepare-styles/colors.scss';
+@import '/src/prepare-styles/mixins.scss'; 
+@import '/src/prepare-styles/usable-fonts.scss'; 
+
+$tablet-min-width: 640px;
+$desktop-min-width: 1200px;
+
+
+
+@mixin content-padding-inline {
+  padding-inline: 16px;
+
+  @include on-tablet {
+    padding-inline: 32px;
+  }
+
+  @include on-desktop {
+    padding-inline: 152px;
+  }
+}
+
+.container {
+  font-family: 'Mont-Regular'; 
+  @include content-padding-inline;
+}
+
+.footer {
+  position: relative;
```

Comment:
Honestly, I don`t understand why you use position relative(if i remember right, it`s default)